tests/callbacks/test_EnqueuedEventTimer.cpp

```diff
@@ -40,26 +42,51 @@ struct EnqueuedEventTimerTest : public ::testing::Test
     timer_t timer {};
 };
 
-//! @test Check @ref Kokkos::utils::callbacks::EnqueuedEventTimer.
-TEST_F(EnqueuedEventTimerTest, duration)
+using TimerTypes = ::testing::Types<
+    EnqueuedEventTimer<execution_space>,
+    EnqueuedEventWithLaunchTimer<execution_space>
+>;
+
+TYPED_TEST_SUITE(EnqueuedEventTimerTest, TimerTypes);
+
+/**
+ * @test Check the duration reported by the timer.
+ *
+ * For @ref Kokkos::utils::callbacks::EnqueuedEventWithLaunchTimer, also check the launch time.
+ */
+TYPED_TEST(EnqueuedEventTimerTest, duration)
```

Comment:
Let's separate this test. But keep the abort one as a typed test.

Reply:
OK. Good suggestion indeed!